tests/e2e/testdata/scenarios/gkehubfeaturemembership/iam_test/script.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+WRITE-KUBE-OBJECT: false
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: mock-project
+  name: containercluster-${uniqueId}
+spec:
+  location: us-central1-a
+  workloadIdentityConfig:
+    workloadPool:  mock-project.svc.id.goog
+---
+WRITE-KUBE-OBJECT: false
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeature
+metadata:
+  name: gkehubfeature-${uniqueId}
+spec:
+  projectRef:
+    external: projects/mock-project
+  location: global
+  resourceID: configmanagement
+---
+WRITE-KUBE-OBJECT: false
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubMembership
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: mock-project
+  name: gkehubmembership-${uniqueId}
+spec:
+  location: global
+  authority:
+    issuer: https://container.googleapis.com/v1/projects/mock-project/locations/us-central1-a/clusters/containercluster-${uniqueId}
+  description: A sample GKE Hub membership
+  endpoint:
+    gkeCluster:
+      resourceRef:
+        name: containercluster-${uniqueId}
+---
+WRITE-KUBE-OBJECT: false
```

Comment:
Why `WRITE-KUBE-OBJECT: false`?  Simply because it's not directly relevant to this test?

If that's the only reason, I sort of like it (the test is more focused) and sort of don't (why not get more coverage if it's "free").  I'm not sure, cc @yuwenma

Reply:
I removed the `WRITE-KUBE-OBJECT: false`. My original thinking was to limit the normalization work to just the resource I am working on.